kudu/src/index.ts

```diff
@@ -0,0 +1,7 @@
+/*---------------------------------------------------------------------------------------------
```

Comment:
Likewise - why are there changes in the kudu folder for this PR? Please revert

Reply:
I keep running into this issue and honestly have no idea why it's happening 😕